core/build.sh

```diff
@@ -38,16 +38,22 @@ case $BUILDER_OS in
       ":x64    64-bit Windows (x64) build"
     )
     ;;
-  mac|linux)
+  mac)
     archtargets+=(
-      ":arch   Linux or mac build -- current architecture"
+      ":mac           Mac all architectures fat library build"
+      ":mac-x86_64    Mac Intel build"
+      ":mac-arm64     Mac arm64 (M1) build"
```

Comment:
Could you elaborate why mac couldn't have 
```
  archtargets+=(
    ":x86    32-bit Mac (x86) build"
    ":x64    64-bit Mac (x64) build"
   ) 
```

Or will it mean later on there will also be
```
":linux-x86_64   "
":linux-arm64    "
```

Reply:
Mac architectures are x86_64 (Intel) and arm64 (M1) -- not x86 (Intel) and x64 (Intel -- technically AMD/Intel). Mac needs a single static library that contains both architectures, and the linker chooses the appropriate architecture to link in.

I don't think we need the same breakdown on Linux at present -- each architecture is built separately and there is no need to provide two architectures in the single distributable.